model/integration_action_test.go

```diff
@@ -24,6 +24,7 @@ func TestTriggerIdDecodeAndVerification(t *testing.T) {
 	t.Run("should succeed decoding and validation", func(t *testing.T) {
 		userId := NewId()
 		clientTriggerId, triggerId, err := GenerateTriggerId(userId, key)
+		assert.Nil(t, err)
```

Comment:
This should be `require`, because the test should not continue if this errors.

Reply:
Done